src/js/game/LevelMVC/LevelView.js

```diff
@@ -1012,17 +1012,33 @@ module.exports = class LevelView {
   }
 
   createActionPlaneBlock(position, blockType) {
-    let blockIndex = (this.yToIndex(position[1])) + position[0];
+    const block = new LevelBlock(blockType);
+    const blockIndex = (this.yToIndex(position[1])) + position[0];
 
     // Remove the old sprite at this position, if there is one.
     this.actionGroup.remove(this.actionPlaneBlocks[blockIndex]);
     this.groundGroup.remove(this.actionPlaneBlocks[blockIndex]);
```

Comment:
What makes this not remove the block beneath rails by accident?  It's a no-op when the target sprite isn't a member of the `groundGroup`?

Reply:
precisely! `group`s are set-style containers